diff --git a/0x02-functions_nested_loops/0-putchar.c b/0x02-functions_nested_loops/0-putchar.c
new file mode 100644
index 0000000..7bf30d7
--- /dev/null
+++ b/0x02-functions_nested_loops/0-putchar.c
@@ -0,0 +1,21 @@
+#include <main.h>
+/**
+ * main - entry point
+ *
+ * Return: 0 Always (success)
+ */
+int main (void)
+{
+         char *sh = "_putchar";
+
+	 while (*sh)
+	 {
+		_putchar(*sh)
+		
+		*sh++;
+	 }
+	 _putchar('\n')
+	
+
+	 return (0);
+}
