diff --git a/0x01-variables_if_else_while/0-positive_or_negative.c b/0x01-variables_if_else_while/0-positive_or_negative.c
old mode 100755
new mode 100644
index fd98f4b..c2ab85a
--- a/0x01-variables_if_else_while/0-positive_or_negative.c
+++ b/0x01-variables_if_else_while/0-positive_or_negative.c
@@ -1,30 +1,47 @@
-#include <stdlib.h>
-#include <time.h>
-#include <stdio.h>
-/**
-* main - Determine if a random number is positive, negative or zero.
- (*
-* Return: 0 on success
-*/
-int main(void)
-{
-	int n;
-
-	srand(time(0));
-	n = rand() - RAND_MAX / 2;
-	/* your code goes there */
-	if (n < 0)
-	{
-		printf("%d is %s\"n", n, "negative");
-	}
-	else if (n > 0)
-	{
-		printf("%d is %s\"n", n, "positive");
-	}
-	else
-	{
-		printf("%d is %s\"n", n, "zero");
-	}
-	return (0);
-
-}
+#include <stdio.h>
+
+#include <stdlib.h>
+
+#include <time.h>
+
+/**
+
+ * main - Entry point
+
+ *
+
+ * Return: Always 0 (Success)
+
+ */
+
+int main(void)
+
+{
+
+	int n;
+
+
+
+	srand(time(0));
+
+	n = rand() - RAND_MAX / 2;
+
+
+
+	if (n > 0)
+
+		printf("%d is positive\n", n);
+
+	else if (n == 0)
+
+		printf("%d is zero\n", n);
+
+	else
+
+		printf("%d is negative\n", n);
+
+
+
+	return (0);
+
+}
diff --git a/0x01-variables_if_else_while/1-last_digit.c b/0x01-variables_if_else_while/1-last_digit.c
old mode 100644
new mode 100755
